MC/config/analysis_testing/json/analyses_config.json

```diff
@@ -96,6 +96,34 @@
                       "o2-analysis-pid-tpc-base",
                       "o2-analysis-pid-tpc-full"]
         },
+        {
+          "name": "TPCSkimming",
+          "enabled": true,
+          "expected_output": [
+            "AnalysisResults.root",
+            "AnalysisResults_trees.root"
+          ],
+          "config": {
+            "data": "json://${O2DPG_ROOT}/MC/config/analysis_testing/json/analysis-skimming.json",
```

Comment:
I suggest to rename to `analysis-testing-TPCSkimming.json` to align with the other configs we have